lab-davide/hello.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
```

Comment:
this project has great file structure :)
seporating things into **lib** and **test** directorys may seem silly now, but make a big diffences in project organization when things start getting more complex !